getting-started/jdbc/README.md

```diff
@@ -37,6 +37,7 @@ This example requires `jq` to be installed on your machine.
 2. Start the docker compose group by running the following command from the root of the repository:
 
     ```shell
+    export PG_JDBC_URL=jdbc:postgresql://postgres:5432/POLARIS
```

Comment:
Should we also export username and password outside of the docker composer? Maybe just me, but feel a bit weird when we allow users to input URL while it has the fixed username and password. 
```
      - QUARKUS_DATASOURCE_USERNAME=postgres
      - QUARKUS_DATASOURCE_PASSWORD=postgres
```

Reply:
we can, didn't change this as both the cloud providers script and getting started uses the same name, l am fine in setting it explicitly